packages/devtools/lib/src/debugger/console_area.dart

```diff
@@ -31,6 +32,9 @@ class ConsoleArea implements CoreElementView {
       const Duration(milliseconds: 100), const Duration(seconds: 1));
   final StringBuffer _bufferedText = StringBuffer();
 
+  /// Ansi terminal color code decoder.
+  AnsiUp _ansiUp = AnsiUp();
```

Comment:
[nit] How about `var _ansiUp = AnsiUp();`

Reply:
If this was a variable within a method I would agree. As this is a non-final member field of a class I think providing the type annotation makes sense.

Reply:
It's not as strongly discouraged as with locals, but initialized fields are allowed to omit the type on the left. Increasingly internally that's the style I see. The name is already present twice, a third copy isn't very useful. Functionally there will be no difference.